### PR TITLE
Create my profile setup

### PR DESCRIPTION
**Pull request by jonas-45**
In this milestone, My profile view was added with the following configurations;

- [ ] React router was used pages navigation routing
- [ ] Navlink component was used to create the navigation links
- [ ] An active class was applied to underline active Navlink

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+.nav-item {
+  cursor: pointer;
+  text-decoration: none;
+  margin-right: 20px;
+  font-size: 1rem;
+}
+
+.separator {
+  margin-right: 10px;
+}
```
Comment (rica213) [inline src/App.css:25]: I'd suggest you use react bootstrap for the styling. Everything here can be done with Bootstrap. This is a good opportunity to learn it.

**Review by rica213 — CHANGES_REQUESTED** [review]
Hi @jonas-45 , 
You have done a really good job. I'm impressed with your speed 😁 
I only have one required change. Please check the comment bellow.

Happy coding

**Review by rica213 — APPROVED** [review]
I'm approving these changes. The remarks I made about the React Bootstrap are optional. We can continue with our project and come back with the styling later on.
Happy coding!

**Comment by jonas-45** [comment]
Thanks @rica213 for your review. I will actually convert these styles to react-bootstrap when the app functionality is done.

**Pull request merged**